packaging/debian/rules

```diff
@@ -20,12 +20,15 @@ PKGDIR = debian/storagenode
 
 override_dh_auto_install:
 # downloaded binaries directory
-	mkdir -p ${PKGDIR}/var/storagenode
+	mkdir -p ${PKGDIR}/var/lib/storagenode
+	mkdir -p ${PKGDIR}/etc/storagenode
 	dh_installsystemd --name=storagenode
```

Comment:
These lines are probably only necessary because we are using debhelper compat level 10 (from the file `debian/compat`). Do we need to do that? I would think all the dists we are targeting support level 12 by now. If we switch to compat level 12, these `dh_installsystemd` calls will happen automatically.

Reply:
@Fadila82 do you know if can change it to 12?

Reply:
This would probably involve:

* remove `debian/compat` file
* change the `Build-Depends` line in `debian/control` to `Build-Depends: debhelper-compat (= 12)`

Reply:
I'm not sure it will automatically install both systemd services, will it? (I'm thinking more specifically about the auto-updater service, I think it is looking for "_package_.service").  https://manpages.debian.org/testing/debhelper/dh_installsystemd.1.en.html

But I'm good with using compat level 12.

Reply:
No, you're right, it will need at least one override line still. I was thinking there were two packages, named storagenode and storagenode-updater, but I was wrong again.

But with compat=12 you should be able to do it like

```#!make
override_dh_installsystemd:
        dh_installsystemd --name=storagenode
        dh_installsystemd --name=storagenode-updater

which is only a little bit better, but avoids confusion about why it's being done in the rule usually reserved for installing binaries into a package.